server/bot/commands/trivia.ts

```diff
@@ -0,0 +1,104 @@
+      if (+args.find(element => element.includes('--time-limit=')).split('=')[1] > 4) {
+        timeLimit = +args.find(element => element.includes('--time-limit=')).split('=')[1]
+      } else {
+        return 'Invalid usage'
+      }
+    }
+    if (args.find(element => element.includes('--max-score='))) {
+      if (+args.find(element => element.includes('--max-score=')).split('=')[1] > 0) {
+        maxScore = +args.find(element => element.includes('--max-score')).split('=')[1]
+      } else {
+        return 'Invalid usage.'
+      }
+    }
+    if (args.find(element => element.includes('--reveal-answer='))) {
+      if (args.find(element => element.includes('--reveal-answer=')).split('=')[1] === 'false') {
+        revealAnswer = false
+      } else if (args.find(element => element === '--reveal-answer=').split('=')[1] !== 'true') {
+        return 'Invalid usage.'
+      }
+    }
+    if (args.length === 1 && ['scores', 'score', 'leaderboard'].includes(args[0])) {
+      const session = tempDB.trivia[message.channel.id]
+      if (session) {
+        message.channel.createMessage(await session.sendScores())
+      } else {
+        return 'There is no trivia session ongoing in this channel.'
+      }
+    } else if (args.length === 1 && args[0] === 'list') {
+      return 'Available trivia lists: \n\n-anime\n-artandliterature\n-cars\n-computers\n-disney\ndota2abilities\n-dota2items\n-elements\n-entertainment\n-finalfantasy\n-gameofthrones\n-games\n-general\n-greekmyth\n-harrypotter\n-leagueoflegends\n-leagueults\n-nba\n-overwatch\n-pokemon\n-slogans\n-sports\n-starwars\n-uscapitals\n-usflags\n-usmap\n-usstateabbreviations\n-warcraft\n-whosthatpokemon\n-worldcapitals\n-worldflgs\n-worldmap'
+    } else if (args.length === 1 && args[0] === 'stop') {
+      const session = tempDB.trivia[message.channel.id]
+      const channel = message.channel as GuildTextableChannel
+      if (session) {
+        if (message.author === session.author || channel.permissionsOf(message.author.id).has('manageMessages')) {
+          await session.endGame()
+          return 'Trivia stopped.'
+        } else {
+          return 'You are not authorized to do that.'
+        }
+      } else {
+        return 'There is no trivia session ongoing in this channel.'
+      }
+    } else if (args[0] !== ('scores' || 'score' || 'leaderboard' || 'stop' || 'list')) {
+      const session = tempDB.trivia[message.channel.id]
+      if (!session) {
+        let triviaList
+        try {
+          triviaList = await parseTriviaList(args[0])
+        } catch (err) {
+          return 'That trivia list doesn\'t exist.'
+        }
+        const t = new TriviaSession(triviaList, message, botPlays, timeLimit, maxScore, revealAnswer, tempDB, client)
+        tempDB.trivia[message.channel.id] = t
+        t.newQuestion()
+      } else {
+        return 'A trivia session is already ongoing in this channel.'
+      }
+    } else {
+      return 'Invalid usage.'
```

Comment:
Add the correct usage after this, `Invalid usage.` was done away with throughout the rest of the bot since it's unhelpful.

Reply:
Also, this won't even get called so you can just remove it.

Reply:
I left this in for cases where the user may try calling the command with multiple arguments (or similar)